src/components/vmd-appointment-map.component.ts

```diff
@@ -2,6 +2,7 @@ import { css, unsafeCSS, customElement, html, LitElement, property, internalProp
 import { Icon, map, marker, tileLayer, LatLngTuple } from 'leaflet';
 import leafletCss from 'leaflet/dist/leaflet.css';
 import leafletMarkerCss from 'leaflet.markercluster/dist/MarkerCluster.Default.css';
+// @ts-ignore
 import { MarkerClusterGroup } from 'leaflet.markercluster';
```

Comment:
Sur mon poste pour pouvoir utiliser le type, j'ai dû utiliser ça : 
```suggestion
import * as L from 'leaflet';
import 'leaflet.markercluster';
```
puis dans le reste du code : 
remplacer `MarkerClusterGroup` par `L.MarkerClusterGroup`

Reply:
Merci pour l'update, j'avais simplement repris la notation de `vmd-lieux.view.ts`
https://github.com/CovidTrackerFr/vitemadose-front/blob/6543d59d1a8666089c5cc616a1b21b4416226968/src/views/vmd-lieux.view.ts#L5-L6

J'ai update la PR